src/Medusa.ts

```diff
@@ -28,69 +28,67 @@ const logger = Logger.contextualize('Medusa');
  * Load medusa and apply all components
  */
 export class Medusa {
-	readonly #express: Express;
-	readonly #rootDir: string;
+    readonly #express: Express;
+    readonly #rootDir: string;
 
-	/**
-	 * @param rootDir Directory where the `medusa-config` is located
-	 * @param express Express instance
-	 */
-	constructor(rootDir: string, express: Express) {
-		this.#express = express;
-		this.#rootDir = rootDir;
-	}
+    /**
+     * @param rootDir Directory where the `medusa-config` is located
+     * @param express Express instance
+     */
+    constructor(rootDir: string, express: Express) {
+        this.#express = express;
+        this.#rootDir = rootDir;
+    }
 
-	/**
-	 * @param modules The modules to load into medusa
-	 */
-	public async load(modules: Constructor<unknown>[]): Promise<AwilixContainer> {
-		const moduleComponentsOptions = metadataReader(modules);
-		const { configModule } = getConfigFile(this.#rootDir, 'medusa-config') as {
-			configModule: {
-				monitoring: MonitoringOptions;
-			};
-		};
+    /**
+     * @param modules The modules to load into medusa
+     */
+    public async load(modules: Constructor<unknown>[]): Promise<AwilixContainer> {
+        const moduleComponentsOptions = metadataReader(modules);
+        const { configModule } = getConfigFile(this.#rootDir, 'medusa-config') as {
+            configModule: {
+                monitoring: MonitoringOptions;
+            };
+        };
 
-		if (configModule.monitoring) {
-			await loadMonitoringModule(this.#express, configModule.monitoring);
-		}
+        await loadMonitoringModule(configModule, this.#express, configModule.monitoring);
 
-		await validatorsLoader(moduleComponentsOptions.get('validator') ?? []);
-		await overrideEntitiesLoader(moduleComponentsOptions.get('entity') ?? []);
-		await overrideRepositoriesLoader(moduleComponentsOptions.get('repository') ?? []);
-		await adminApiLoader(
-			this.#express,
-			moduleComponentsOptions.get('middleware') ?? [],
-			moduleComponentsOptions.get('router') ?? []
-		);
-		await storeApiLoader(
-			this.#express,
-			moduleComponentsOptions.get('middleware') ?? [],
-			moduleComponentsOptions.get('router') ?? []
-		);
-		await databaseLoader(
-			moduleComponentsOptions.get('entity') ?? [],
-			moduleComponentsOptions.get('repository') ?? []
-		);
-		await pluginsLoadersAndListeners(this.#express);
-		await servicesLoader(moduleComponentsOptions.get('service') ?? []);
+        await validatorsLoader(moduleComponentsOptions.get('validator') ?? []);
+        await overrideEntitiesLoader(moduleComponentsOptions.get('entity') ?? []);
+        await overrideRepositoriesLoader(moduleComponentsOptions.get('repository') ?? []);
+        await adminApiLoader(
+            this.#express,
+            moduleComponentsOptions.get('middleware') ?? [],
+            moduleComponentsOptions.get('router') ?? []
```

Comment:
Codacy has a fix for the issue: [Replace `············` with `↹↹↹`](https://app.codacy.com/gh/adrien2p/medusa-extender/pullRequest?prid=9061797)
```suggestion
			moduleComponentsOptions.get('router') ?? []
```